docs/karpor/4-developer-guide/1-contribution-guide/2-code-contribute.md

```diff
@@ -15,19 +15,19 @@ This guide will help you get started with Karpor development.
 
 ### Prerequisites
 
-* Golang version 1.19+
+* Golang version 1.22+
 
 <details>
   <summary>Installing Golang</summary>
 
-1. Install go1.19+ from the [official website](https://go.dev/dl/). Extract the binary files and place them at a location, assuming it is located under the home directory `~/go/`, here is an example command, you should choose the correct binary file for your system.
+1. Install go1.22+ from the [official website](https://go.dev/dl/). Extract the binary files and place them at a location, assuming it is located under the home directory `~/go/`, here is an example command, you should choose the correct binary file for your system.
 
 ```
 wget https://go.dev/dl/go1.20.2.linux-amd64.tar.gz
```

Comment:
Please also update this example command.

Reply:
Nice catch👍